nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/app.module.ts

```diff
@@ -56,7 +57,13 @@ import { MatDialogModule } from '@angular/material/dialog';
             routerState: RouterState.Minimal,
             navigationActionTiming: NavigationActionTiming.PostActivation
         }),
-        EffectsModule.forRoot(UserEffects, ExtensionTypesEffects, AboutEffects, StatusHistoryEffects),
+        EffectsModule.forRoot(
+            UserEffects,
+            ExtensionTypesEffects,
+            AboutEffects,
+            StatusHistoryEffects,
+            EnableControllerServiceEffects
```

Comment:
This name, `EnableControllerServiceEffects`, gives the impression the effects are only useful for enabling controller services. However, all of the disable-related effects are defined in there as well.
Can we find a more appropriate name? I suppose this would include the actions, reducers, and selectors as well.